jose-jws/src/crypto/mod.rs

```diff
@@ -3,8 +3,6 @@
 
 //! JWS Cryptographic Implementation
 
-mod core;
```

Comment:
I'm not sure what was the purpose of this, but the traits defined there seem to be unused duplicates of the ones already defined here.